### PR TITLE
Clean the file structure a little

### LINKED ISSUE #19: Update file structure
After cloning and contributing to the repo, I think it would be good to rearrange the files (for example have a folder front), in order to make the code more readable and more maintainable (there are starting to be a lot of lines in the files, which makes them more complex to understand)

_Originally posted by @Queng123 in https://github.com/cryck/sticker-composer/issues/18#issuecomment-1940978428_

### PR DESCRIPTION
**Pull request by cryck**
A little bit of cleanup
Fixes #19

### CODE REVIEW

**Review by Queng123 — APPROVED** [review]

**Comment by cryck** [comment]
@Queng123 yeah I agree, I let this be on hold for the moment until we merged in the canvas. Afterwards I would update this PR to also have the other files moved to a better location.

**Comment by cryck** [comment]
Solved in https://github.com/cryck/sticker-composer/pull/53

**Pull request closed without merging**
